### DISCUSSION TITLE
Can't Close Restart or Logout of MacOS after Installing Plugin  i.m Stuck on Finder Closed no Top Menu Bar (Solved)

### DISCUSSION THREAD

**Discussion by Linuxvst** [apohl79/audiogridder#909] [Help]
Hi i Just Found About this Project Interesting!

here is my Problem !
After installing the Plugin on Mac OS El capitan i am not able to Shut Down, Restart or Logout 
here is what is Happening 
it Closes the Finder (no More top Menu Bar)
i just have the Desktop image and Dock 
i have to use the Power Button to Shutdown 

 after i rebooted and the only way i found to Shut down the OS was with a terminal Command

i Tried Both Versions of the Plugin 
the Stable Release (https://audiogridder.com/releases/AudioGridderPlugin_1.1.1_macOS-x86_64.pkg)

and The Beta Version (https://audiogridder.com/releases/AudioGridderPlugin_1.2.0-beta6_macOS-x86_64.pkg)

i Tried by Removing all files related by checking the installer with Pacifist 
i removed them manually but no Success it still does not Shutdown Restart or Logout!
i.ve noticed that there is a Script running at the end of the installer to force the use of all the cpu cores could this be the problem?
i have 3 macs 
1 Mac Pro 3.1 with El Capitan & Catalina 
and 2 Mac Pros 5.1  With Mac OS Catalina, Big Sur &  Monterey installed 

 P.S. i had to restore all my System with a Backup i Gave up after trying 3 Times
 (i.m already using Hise and Juce )

**Comment by oldtos** [comment]
https://discussions.apple.com/thread/8300811
https://appletoolbox.com/mac-will-not-shut-fix/

**Reply by Linuxvst** [reply]
i Know all About this it is not a SMC or Pram Reset that Fixes the Problem 
i already tried!
Something is actually Changed on the OS after Installing the Plugin i Can't point out were the change was Made
after Checking .plist Files and Looking into the installer with Pacifist to see what was installed 
also looking in the "Launch Agents" and "Launch Daemons" i did not find anything related ! 
i think it has to do with the Plugin Tray App not unloading when we try to Restart Shutdown or Logout  it stalls the system
 it's probably waiting for that Plugin Tray App to Unload!

**Reply by Linuxvst** [reply]
i Finally got a Way to Make it Work i just Downloaded the Binaries and Copied them to the proper locations 
there is Probably Something wrong with the Installer i think it's the script running at the end to open all cores 
it Probably leaves something Open!
this is probably why it does nor Shutdown Restart or logout if you  use the installer
so if anybody come across this Problem Just Download The Binaries and copy them
